lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -198,7 +198,9 @@ sub _tag_helpers {
   return $assets->grep(sub { !$_->isa('Mojolicious::Plugin::AssetPack::Asset::Null') })
     ->map(
     sub {
-      my $tag_helper = $_->format eq 'js' ? 'javascript' : 'stylesheet';
+      my $tag_helper = $_->format eq 'js'  ? 'javascript' :
+                       $_->format eq 'png' ? 'image'      :
+                                             'stylesheet';
```

Comment:
I don't think that's a big deal. I can see that it's ugly doing this from the outside, but I'm not sure how many more there could be after adding "image" to the mix.